src/middleware.ts

```diff
@@ -1 +1,66 @@
-export function middleware(): void {}
+import { EventEmitter } from 'events'
+import { Headers } from 'headers-utils'
+import { RequestHandler as ExpressMiddleware } from 'express'
+import { RequestHandler, handleRequest, parseIsomorphicRequest } from 'msw'
+
+const emitter = new EventEmitter()
```

Comment:
Should this be exposed at all to the user if they want to listen to events for some reason? Or is it basically a no-op?

Reply:
I don't think we need to expose this emitter to the user. It's needed internally to enable life-cycle events in MSW. It's a low-level API that is not public: users consume life-cycle events through `setupWorker`/`setupServer` and there is neither of those API being used to establish a middleware.